### PR TITLE
feat(atom/button): add elevation prop

### PR DESCRIPTION
**Pull request by andresz1**
## atom/button
**TASK**: None

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description, Motivation and Context
Add `elevation` prop to button

### Screenshots - Animations
Screenshot 2020-10-12 at 21 28 59

### CODE REVIEW

File: components/atom/button/src/index.scss [hunk 1]
```diff
@@ -240,6 +251,11 @@ $icon-stroke-color: currentColor !default;
     $color-invert: nth($pair, 2);
 
     &--#{$name}Color {
+      &:focus {
+        box-shadow: 0 0 4px 1px rgba($color, 0.6),
+          0 -8px 12px -6px rgba($color, 0.5);
```
Comment (turolopezsanabria) [inline components/atom/button/src/index.scss:256]: Please add a `default!` variable here, thnx!

**Review by turolopezsanabria — COMMENTED** [review]
As mentioned in the weekly, I think this adjustment is valid to be added at company level, and could apply to all verticals and themes.

I've added the label accordingly

**Review by andresin87 — APPROVED** [review]

**Comment by turolopezsanabria** [comment]
yey, helloooo 
I think this one was ok to move forward, right?

**Comment by turolopezsanabria** [comment]
hello @andresz1 
Do you need help to progress with this PR?

**Pull request closed without merging**